cmd/tag-write.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	analytics "github.com/segmentio/analytics-go/v3"
+	"github.com/segmentio/chamber/v2/store"
+	"github.com/segmentio/chamber/v2/utils"
+	"github.com/spf13/cobra"
+)
+
+var (
+	deleteOtherTags bool
+
+	// tagWriteCmd represents the tag read command
+	tagWriteCmd = &cobra.Command{
+		Use:   "write <service> <key> <tag>...",
+		Short: "Write tags for a specific secret",
+		Args:  cobra.MinimumNArgs(3),
+		RunE:  tagWrite,
+	}
+)
+
+func init() {
+	tagWriteCmd.Flags().BoolVar(&deleteOtherTags, "delete-other-tags", false, "Delete tags not specified in the command")
```

Comment:
This is a descriptive variable name, but I'm curious if there is precedent for CLIs to have an argument that is semantically the same but more conventional? Maybe `--reset` or `--sync` or `--prune`? Really not asking for a change just opening up a discussion about what a conventional flag someone might expect to exist on a CLI that has this behavior in a low surprise manner

Reply:
I didn't really do a survey, but I did ask OpenAI for suggestions and chose this 😆

Reply:
Hah! Funny. I asked open AI the same thing and it suggested those (specifically sync and prune) 😂

Reply:
I guess it depends on the question phrasing and OpenAI's mood that day!